frontend/services/screenshot_service/Screenshot_Service.cpp

```diff
@@ -262,6 +275,20 @@ void Screenshot_Service::postGraphRender() {
     // swap buffers, glClear
 }
 
+/// XXX TEMP
+bool Screenshot_Service::guiRenderCallback(WinConfig_t& wc) {
+    bool valid_imgui_scope =
+            ((ImGui::GetCurrentContext() != nullptr) ? (ImGui::GetCurrentContext()->WithinFrameScope) : (false));
```

Comment:
Is there some way to automate this validity check, e.g. via a helper function one can call?

Reply:
see commit b3fe3c4